fern-logger/src/config.rs

```diff
@@ -63,6 +63,18 @@ impl LoggerOutputConfigBuilder {
         self
     }
 
+    /// Sets a collection of exclusions of a logger output.
+    /// A message is logged only if one of the exclusions is *not* part of the log's metadata target.
+    pub fn target_exclusions(mut self, target_exclusions: &[&str]) -> Self {
+        self.target_exclusions = Some(
+            target_exclusions
+                .iter()
+                .map(ToString::to_string)
```

Comment:
It triggers me that the impl is slightly different than `target_filters`, can you harmonise please ? To whatever you prefer.

Reply:
Ha, yes that's fine :smile: